diff -U3 -r kernel.orig/nv-drm.c kernel/nv-drm.c
--- a/kernel/nv-drm.c	2019-12-25 15:56:36.810442352 +0300
+++ b/kernel/nv-drm.c	2019-12-25 15:57:04.003551618 +0300
@@ -173,9 +173,7 @@
 static struct drm_driver nv_drm_driver = {
 
     .driver_features = DRIVER_GEM
-#if defined(NV_DRM_DRIVER_LEGACY_FEATURE_BIT_PRESENT)
         | DRIVER_LEGACY
-#endif
 #if defined(NV_DRM_DRIVER_PRIME_FLAG_PRESENT)
         | DRIVER_PRIME
 #endif
